src/views/desktop/Desktop.tsx

```diff
@@ -1,16 +1,18 @@
-import { useEffect } from 'preact/hooks';
+import { useEffect, useRef } from 'preact/hooks';
 import { StartupChime } from '__/components/Desktop/StartupChime';
 import { WindowsArea } from '__/components/Desktop/Window/WindowsArea';
 import { Dock } from '__/components/dock/Dock';
 import { TopBar } from '__/components/topbar/TopBar';
 import { useTheme } from '__/hooks';
 import css from './Desktop.module.scss';
+import ContextMenu from '__/components/Desktop/ContextMenu/ContextMenu';
 
 const DarkBackground = '/assets/wallpapers/3-1.jpg';
 const LightBackground = '/assets/wallpapers/3-2.jpg';
 
 export const Desktop = () => {
   const [theme] = useTheme();
+  const outerRef: React.LegacyRef<HTMLElement> | undefined = useRef(null);
```

Comment:
Hi, I see you typed the useRef manually. Is there a reason not to use `useRef<HTMLDivElement>()` and this instead?

Reply:
Nah no reason for that it's just that I have been using that, switched to `useRef<HTMLDivElement>()`